client/my-sites/backup/backup-storage-space/backup-storage-space-upsell/style.scss

```
@@ -0,0 +1,42 @@
+.backup-storage-space-upsell__title-text {
+	font-size: 1rem;
+	font-weight: bold;
+	margin-bottom: 16px;
+}
+
+a.backup-storage-space-upsell__button {
+	color: #1e1e1e;
+}
+
+.backup-storage-space-upsell__button {
+	border: 2px solid var( --studio-jetpack-green );
+	border-radius: 2px;
+	padding: 16px 24px;
+	width: 100%;
+	height: 100%;
+	font-size: 1rem;
```

Another one for @Automattic/jetpack-design to confirm, but I think they generally prefer we use SCSS variables for font sizes:

```suggestion
	font-size: $font-body;
```